cmd/lookoutd/web.go

```diff
@@ -21,12 +21,17 @@ type WebCommand struct {
 	ServerURL          string `long:"server" env:"LOOKOUT_SERVER_URL" description:"URL used to access the web server in the form 'HOSTNAME[:PORT]'. Leave it unset to allow connections from any proxy or public address"`
 	FooterHTML         string `long:"footer" env:"LOOKOUT_FOOTER_HTML" description:"Allows to add any custom html to the page footer. It must be a string encoded in base64. Use it, for example, to add your analytics tracking code snippet"`
 
+	ClientID     string `long:"client-id" env:"LOOKOUT_CLIENT_ID" required:"true" description:"GitHub App OAuth credentials"`
```

Comment:
one-line-struct-tag-in-go is a [pitty](https://github.com/golang/go/issues/15893) :(